notifier/operator/templates/base.mjml.j2

```diff
@@ -0,0 +1,58 @@
+<mjml>
+  <mj-body>
+    <!-- Company Header -->
+    <mj-section full-width="full-width" background-color="#151515" padding-top="8px" padding-bottom="8px" padding-left="12px" padding-right="12px">
+      <mj-group>
+        <mj-column vertical-align="middle">
+          <mj-image href="https://demo.redhat.com/" title="RedHat" alt="RedHat" width="185px" src="https://public-assets-email.s3.us-east-2.amazonaws.com/Logo-Red_Hat-A-Reverse-RGB.png" />
+        </mj-column>
+      </mj-group>
+    </mj-section>
+    <!-- spacer -->
+    <mj-section padding-top="0" padding-bottom="0" padding-left="0" padding-right="0">
+      <mj-column>
+        <mj-spacer height="25px" />
+      </mj-column>
+    </mj-section>
+    <!-- Title -->
+    <mj-section padding-top="21px" padding-bottom="12px" padding-left="12px" padding-right="12px">
+      <mj-column>
+        <mj-text font-family="Red Hat Text" font-weight="regular" align="center" font-size="16px" color="#000" line-height="1.2">
+          Status: Service Ready
```

Comment:
should this be better placed under the content block?

Reply:
Good catch, I've fixed it with 75d1a3bb7e54a6de9db003c978327d6543755d58